### PR TITLE
151 foldable chapters

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

**Review by avirankatz — CHANGES_REQUESTED** [review]
Looks and behaves good! Two minor requests:

- [ ] Every chapter should be expanded by default (`defaultExpanded` prop allows that)
- [ ] Let's use chapter.id as key instead of a random string

File: frontend/src/pages/AgreementPage.tsx [hunk 1]
```diff
@@ -75,40 +88,61 @@ const AgreementPage: FC = () => {
...
-                );
-              })}
-            </Stack>
-          </Stack>
-        ))}
+        <Stack>
+          {agreement?.chapters?.map((chapter: IChapter) => (
+            <Accordion
+              TransitionProps={{ unmountOnExit: true }}
+              key={generateRandString()}
```
Comment (avirankatz) [inline frontend/src/pages/AgreementPage.tsx:95]: Let's use `defaultExpanded` prop here

[the remark this answers is not in the document]
Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:95]: done

Comment (yishain11) [reply frontend/src/pages/AgreementPage.tsx:95]: done

**Review by avirankatz — APPROVED** [review]

**Pull request merged**